Error with release

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
Release works

## Current Behavior
Release fails:
```
2022-10-28T23:15:04.2188193Z To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
2022-10-28T23:15:04.3267798Z /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/response/raise_error.rb:14:in `on_complete': GET https://api.github.com/user: 401 - Bad credentials // See: https://docs.github.com/rest (Octokit::Unauthorized)
2022-10-28T23:15:04.3269136Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/faraday-2.6.0/lib/faraday/middleware.rb:18:in `block in call'
2022-10-28T23:15:04.3270414Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/faraday-2.6.0/lib/faraday/response.rb:42:in `on_complete'
2022-10-28T23:15:04.3271402Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/faraday-2.6.0/lib/faraday/middleware.rb:17:in `call'
2022-10-28T23:15:04.3272401Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/middleware/follow_redirects.rb:73:in `perform_with_redirection'
2022-10-28T23:15:04.3273398Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/middleware/follow_redirects.rb:61:in `call'
2022-10-28T23:15:04.3274357Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/faraday-2.6.0/lib/faraday/rack_builder.rb:153:in `build_response'
2022-10-28T23:15:04.3275568Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/faraday-2.6.0/lib/faraday/connection.rb:445:in `run_request'
2022-10-28T23:15:04.3276458Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/faraday-2.6.0/lib/faraday/connection.rb:200:in `get'
2022-10-28T23:15:04.3277309Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/sawyer-0.9.2/lib/sawyer/agent.rb:99:in `call'
2022-10-28T23:15:04.3278180Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/connection.rb:156:in `request'
2022-10-28T23:15:04.3279058Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/connection.rb:19:in `get'
2022-10-28T23:15:04.3279931Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/client/users.rb:34:in `user'
2022-10-28T23:15:04.3280810Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/octokit-6.0.0/lib/octokit/configurable.rb:137:in `login'
2022-10-28T23:15:04.3281472Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/2.7.0/delegate.rb:83:in `method_missing'
2022-10-28T23:15:04.3282321Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/github/client.rb:22:in `login!'
2022-10-28T23:15:04.3283182Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/github.rb:58:in `tap'
2022-10-28T23:15:04.3284046Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/github.rb:58:in `client'
2022-10-28T23:15:04.3284952Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/github.rb:36:in `existing_release'
2022-10-28T23:15:04.3286090Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/github.rb:23:in `create_or_update_release'
2022-10-28T23:15:04.3287515Z Push v9.1.0… ✘
2022-10-28T23:15:04.3288364Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/commands/push.rb:30:in `block in call'
2022-10-28T23:15:04.3290514Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/commands/push.rb:48:in `block (2 levels) in each_tag_with_version_and_notes'
2022-10-28T23:15:04.3291389Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/logger.rb:34:in `benchmark'
2022-10-28T23:15:04.3291969Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/2.7.0/forwardable.rb:235:in `benchmark'
2022-10-28T23:15:04.3292832Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/commands/push.rb:47:in `block in each_tag_with_version_and_notes'
2022-10-28T23:15:04.3293695Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/commands/push.rb:42:in `each'
2022-10-28T23:15:04.3294578Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/commands/push.rb:42:in `each_tag_with_version_and_notes'
2022-10-28T23:15:04.3295564Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/commands/push.rb:29:in `call'
2022-10-28T23:15:04.3296344Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/lib/chandler/cli.rb:22:in `run'
2022-10-28T23:15:04.3297107Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/lib/ruby/gems/2.7.0/gems/chandler-0.7.0/exe/chandler:4:in `<top (required)>'
2022-10-28T23:15:04.3297629Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/bin/chandler:23:in `load'
2022-10-28T23:15:04.3298103Z 	from /opt/hostedtoolcache/Ruby/2.7.6/x64/bin/chandler:23:in `<main>'
2022-10-28T23:15:04.3326326Z ##[error]Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -2,7 +2,6 @@ name: ci
 on: [push]
 
 jobs:
-
   check_mturk_changes:
     runs-on: ubuntu-latest
     outputs:
@@ -72,7 +71,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '10'
+          node-version: "10"
       - name: Set up Ruby
         uses: actions/setup-ruby@v1
         with:
@@ -81,7 +80,6 @@ jobs:
         run: |
           gem install danger
           gem install danger-commit_lint
-          gem install chandler -v 0.7.0
       - name: Yarn
         run: yarn --frozen-lockfile --ignore-engines
       - name: Set up Python ${{ matrix.python-version }}
@@ -180,15 +178,15 @@ jobs:
       - name: Set up Python 3.10
         uses: actions/setup-python@v2
         with:
-            python-version: "3.10"
+          python-version: "3.10"
       - name: Cache pip
         uses: actions/cache@v2
         with:
-            path: ~/.cache/pip
-            key: ${{ runner.os }}-pip-${{ hashFiles('*.txt') }}
-            restore-keys: |
-              ${{ runner.os }}-pip-
-              ${{ runner.os }}-
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
       - name: Install Ubuntu packages
         run: sudo apt-get --yes install enchant curl
       - name: Install and upgrade wheel, pip and tox
@@ -207,10 +205,9 @@ jobs:
         continue-on-error: true
         run: for container in $(docker ps -a -q); do echo "***** $(docker ps -a|grep $container) *******"; docker logs $container; done
 
-
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
-    - uses: pre-commit/action@v2.0.0
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -1,5 +1,5 @@
 name: Release
-on: 
+on:
   push:
     tags:
       - v*
@@ -16,12 +16,10 @@ jobs:
         uses: actions/setup-ruby@v1
         with:
           ruby-version: 2.7
-      - name: Install Chandler
-        run: gem install chandler -v 0.7.0
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '10'
+          node-version: "10"
       - name: Install node packages
         run: yarn --frozen-lockfile --ignore-engines
       - name: Cache pip
@@ -35,13 +33,10 @@ jobs:
       - name: Python Installers
         run: pip install --upgrade pip wheel tox
       - name: Set up deployment
-        env:
-          CHANDLER_GITHUB_API_TOKEN: ${{ secrets.CHANDLER_GITHUB_API_TOKEN }}
         run: |
           pandoc --from=markdown --to=rst --output=README.rst README.md
           pip install build
           python -m build --sdist --wheel .
-          chandler push --tag-prefix=v
       - name: Deploy
         uses: pypa/gh-action-pypi-publish@v1.4.2
         with:

--- a/scripts/github_releases.sh
+++ b/scripts/github_releases.sh
@@ -1,3 +1,0 @@
-echo "machine api.github.com\n  login DallingerBot\n  password $CHANDLER_GITHUB_API_TOKEN" > ~/.netrc
-chmod 0600 ~/.netrc
-chandler push

